velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -1061,9 +1077,16 @@ MaterializedRowMultiset materialize(const std::vector<RowVectorPtr>& vectors) {
 
 bool assertEqualResults(
     const MaterializedRowMultiset& expectedRows,
+    const TypePtr& expectedType,
     const std::vector<RowVectorPtr>& actual) {
+  const auto actualType =
+      (materialize(actual).size() == 0) ? nullptr : actual.at(0)->type();
```

Comment:
Why do we have the materialize the `actual` input here? Can't we simply use
```
const auto actualType =
      (actual.size() == 0) ? nullptr : actual.at(0)->type();
```

Reply:
Right, we don't need materialize here. Removed the same.